includes/wikia/nirvana/WikiaApp.class.php

```diff
@@ -633,6 +636,10 @@ public function sendRequest( $controllerName = null, $methodName = null, $params
 
 		$request = new WikiaRequest($params);
 
+		if ( false === $internal ) {
+			$internal = F::app()->wg->Request->isWikiaInternalAuthorizedRequest();
```

Comment:
@slayful, we still need this change, but it the above line should be changed to the existing header method:
```php
$internal = F::app()->wg->Request-> isWikiaInternalRequest();
```